src/Deployment/Properties/launchSettings.json

```diff
@@ -3,7 +3,11 @@
     "Visual Studio Extension": {
       "commandName": "Executable",
       "executablePath": "$(DevEnvDir)devenv.exe",
-      "commandLineArgs": "/rootsuffix $(VSSDKTargetPlatformRegRootSuffix) /log"
+      "commandLineArgs": "/rootsuffix $(VSSDKTargetPlatformRegRootSuffix) /log",
+      "environmentVariables": {
+        "VsImageTheming_MaxBlackAndWhiteBackgroundLuminosityRange": "0",
+        "VsImageTheming_MinBlackAndWhiteBackgroundLuminosityRange": "0"
```

Comment:
what is this?

Reply:
Oh this was debugging for the blue theme monikers. I need to remove them from my settings.